data/human/news.txt

```diff
@@ -60,6 +60,18 @@ news "merchant anywhere"
 			"The other day I happened across a rock band whose transport flaked out. Good thing I had seats for all eight of them, because it got me into a free concert at their next stop."
 			"Always look for work that provides a lot of detail. The more interesting it sounds, the more likely it is to end up giving you stories to tell your kids someday."
 			"For some reason, boring jobs almost never hang out in the spaceport. So, if you are looking for interesting or at least varied work..."
+			"If you can't stand the heat, stay out of the solar storms."
+			"Silence is golden, except when you are listening to the life support systems."
+			`Even in this day and age some people still say "the sun goes down", instead of the more appropriate "the sun goes behind the horizon".`
```

Comment:
```suggestion
			`Even in this day and age some people still say "the sun goes down," instead of the more appropriate "the sun goes behind the horizon."`
```